src/utils/fetch-icon.ts

```diff
@@ -0,0 +1,14 @@
+export default function fetchSvgIcon(iconUrl: string) {
```

Comment:
Do we really need to do the web browser job?

Reply:
What do you suggest we do here?

Reply:
This won't be necessary as we will include all the icons for a fixed set of applications in the web component itself.